api/plugins/notifications.py

```diff
@@ -135,6 +156,21 @@ def access_expiring_owner(
         logger.exception("Failed to execute access expiring for owner notification callback")
 
 
+@hookimpl(wrapper=True)
+def access_expiring_role_owner(
```

Comment:
Can you add an example implementation in our example notifications plugins?

- https://github.com/discord/access/blob/main/examples/plugins/notifications_slack/notifications.py
- https://github.com/discord/access/blob/main/examples/plugins/notifications/notifications.py